dfget/util/log.go

```diff
@@ -35,7 +35,6 @@ func CreateLogger(logPath string, logName string, logLevel string, sign string)
 	// parse log level
 	level, err := log.ParseLevel(logLevel)
 	if err != nil {
-		// TODO: print a warning.
```

Comment:
I think we should print a warning here? WDYT?

Reply:
`dfget` only use `debug` or `info`, the code here just in case.